sql/sql_plugin.cc

```diff
@@ -4543,6 +4543,8 @@ my_bool post_init_callback(THD *thd, void *)
     set_current_thd(thd);
     plugin_thdvar_init(thd);
 
+    // Restore proper transaction read-only context
+    thd->variables.tx_read_only= false;
```

Comment:
Can you explain why this needs to be here, assuming that tx_read_only=true is configuration  doest this just overwrite it ?